Example/test-suite/tests/Colors.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+import { processColor } from 'react-native-reanimated';
+
+export const name = 'Colors';
+
+export function test(t) {
+  t.describe('colors', () => {
+    t.it('process basic colors', () => {
+      const testData = [
+        ['red', 'ff0000'],
```

Comment:
Does this work? I guess the values should be different for iOS and Android.

Reply:
cdf71f0